core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -975,16 +1140,39 @@ private CarbonCommonConstants() {
    * make it less and not the index writing speed.
    */
   public static final String CARBON_LUCENE_COMPRESSION_MODE_DEFAULT = "speed";
+
   /**
    * The node loads the smallest amount of data
    */
   @CarbonProperty
   public static final String CARBON_LOAD_MIN_SIZE_INMB = "load_min_size_inmb";
+
+  /**
+   * the node minimum load data default value
+   */
   public static final String CARBON_LOAD_MIN_SIZE_INMB_DEFAULT = "0";
+
+  @CarbonProperty
+  public static final String IN_MEMORY_FOR_SORT_DATA_IN_MB = "sort.inmemory.size.inmb";
+
+  public static final String IN_MEMORY_FOR_SORT_DATA_IN_MB_DEFAULT = "1024";
+
+  /**
+   * carbon sort size
+   */
+  @Deprecated
```

Comment:
Seems it is still used in SortDataRows, so it is not deprecated

Reply:
ok